[2023-09-17T18:07:36.089+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.executaCrawler manual__2023-09-17T14:17:43.647960+00:00 [queued]>
[2023-09-17T18:07:36.098+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.executaCrawler manual__2023-09-17T14:17:43.647960+00:00 [queued]>
[2023-09-17T18:07:36.098+0000] {taskinstance.py:1359} INFO - Starting attempt 2 of 2
[2023-09-17T18:07:36.111+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonDecoratedOperator): executaCrawler> on 2023-09-17 14:17:43.647960+00:00
[2023-09-17T18:07:36.115+0000] {standard_task_runner.py:57} INFO - Started process 207 to run task
[2023-09-17T18:07:36.118+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'etl', 'executaCrawler', 'manual__2023-09-17T14:17:43.647960+00:00', '--job-id', '146', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpq3_t3s9t']
[2023-09-17T18:07:36.120+0000] {standard_task_runner.py:85} INFO - Job 146: Subtask executaCrawler
[2023-09-17T18:07:36.165+0000] {task_command.py:415} INFO - Running <TaskInstance: etl.executaCrawler manual__2023-09-17T14:17:43.647960+00:00 [running]> on host 2b62bebe73f6
[2023-09-17T18:07:36.625+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='executaCrawler' AIRFLOW_CTX_EXECUTION_DATE='2023-09-17T14:17:43.647960+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-17T14:17:43.647960+00:00'
[2023-09-17T18:07:36.628+0000] {logging_mixin.py:151} INFO - Executando o crawler...
[2023-09-17T18:07:36.629+0000] {logging_mixin.py:151} INFO - Período de referência: 07/2023
[2023-09-17T18:07:36.629+0000] {logging_mixin.py:151} INFO - Requisitando a Rota: '/json_empenhos'
[2023-09-17T18:07:36.651+0000] {logging_mixin.py:151} INFO - Requisitando a URL: https://trescoracoes-mg.portaltp.com.br/api/transparencia.asmx/json_empenhos?ano=2023&mes=7
[2023-09-17T18:07:37.616+0000] {logging_mixin.py:151} INFO - tipo de dados:  <class 'httpx.Response'>
[2023-09-17T18:07:37.617+0000] {logging_mixin.py:151} INFO - ###########
[2023-09-17T18:07:37.639+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/base.py", line 221, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/etl.py", line 9, in executaCrawler
    main()
  File "/opt/airflow/dags/crawler.py", line 116, in main
    report.run()
  File "/opt/airflow/dags/crawler.py", line 88, in run
    self.insereDados()
  File "/opt/airflow/dags/crawler.py", line 75, in insereDados
    engine = criaEngine(self.__bd)
AttributeError: 'Report' object has no attribute '_Report__bd'
[2023-09-17T18:07:37.650+0000] {taskinstance.py:1398} INFO - Marking task as FAILED. dag_id=etl, task_id=executaCrawler, execution_date=20230917T141743, start_date=20230917T180736, end_date=20230917T180737
[2023-09-17T18:07:37.669+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 146 for task executaCrawler ('Report' object has no attribute '_Report__bd'; 207)
[2023-09-17T18:07:37.713+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-09-17T18:07:37.734+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
